src/styles/index.scss

```
@@ -1,3 +1,89 @@
-body {
+$product-font-color: #060b35;
+
+@import 'stars';
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.product-cards-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.product-card {
+  width: 200px;
+  border-radius: 5px;
```

as previous mentor asked you should add padding to a card
```suggestion
  border-radius: 5px;
  padding: 32px 16px 16px;
```